src/widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -193,8 +182,8 @@ export default function hierarchicalMenu({
   templates = defaultTemplates,
   cssClasses: userCssClasses = {},
 } = {}) {
-  if (!container || !attributes || !attributes.length) {
```

Comment:
This was testing the `attributes` option while it's the connector responsibility.